applications/openshift/api-server/api_server_admission_control_plugin_Scc/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Enable the SecurityContextConstraint Admission Control Plugin'
+
+description: |-
+   To ensure pod permissions are managed, make sure that the
+   <tt>SecurityContextConstraint</tt> admission control plugin is used.
+
+rationale: |-
+  SCCs to define a set of conditions that a pod must run with in order to
+  be accepted into the system.
```

Comment:
That's nice, but what is the security rationale?

Reply:
Fixed locally, will mark as resolved when I push new patches.